javasample/src/main/java/com/oursky/authgeartest/MainViewModel.java

```diff
@@ -261,6 +265,24 @@ public void sendWechatAuthRequest(String state) {
                 req.state = state;
                 wechatAPI.sendReq(req);
             }
+
+            @Override
+            public void onOpenEmailClient(@NonNull Context context) {
+                Intent intent = Authgear.Companion.makeEmailClientIntentChooser(
```

Comment:
Authgear.Companion does not follow Java's convention. Add `@JvmField` to makeEmailClientIntentChooser to make it directly accessible under Authgear.